src/query/trigger.cpp

```diff
@@ -432,6 +432,21 @@ void TriggerStore::DropTrigger(const std::string &name) {
   storage_.Delete(name);
 }
 
+void TriggerStore::DropAll() {
+  std::unique_lock store_guard{store_lock_};
+  std::vector<std::string> trigger_names{};
+  for (auto const &[name, trigger_data] : storage_) {
+    trigger_names.push_back(name);
```

Comment:
Since we know the size of `triger_names` in advance, it might be worth reserving the vector's capacity.

Reply:
Why even have a temporary `triger_names`?
Why not `DeletePrefix("")`?